cl_sii/libs/mm_utils.py

```diff
@@ -99,40 +101,44 @@ def __init__(self, format: Optional[str] = None, **kwargs: Any) -> None:
         self.dateformat = format
 
     def _add_to_schema(self, field_name: str, schema: marshmallow.Schema) -> None:
```

Comment:
The `_add_to_schema()` method was renamed in the superclass to `_bind_to_schema()`.

```suggestion
    def _bind_to_schema(self, field_name: str, schema: marshmallow.Schema) -> None:
```